cw_bitcoin/lib/electrum_derivations.dart

```diff
@@ -4,7 +4,7 @@ Map<DerivationType, List<DerivationInfo>> electrum_derivations = {
   DerivationType.electrum: [
     DerivationInfo(
       derivationType: DerivationType.electrum,
-      derivationPath: "m/0'",
+      derivationPath: "m/0'/0",
```

Comment:
why not add the un-hardened 0 to the other paths as well?

Reply:
I'm not sure what you mean? I did?

Reply:
for the other derivations some of them are without the un-hardened 0